cmd/controller/main.go

```diff
@@ -15,7 +15,11 @@ limitations under the License.
 package main
 
 import (
+	"github.com/bwagner5/kompat/pkg/kompat"
```

Comment:
This is a dependency from an unknown author and this will raise some eyebrows from people in the community. I don't think we can just straightaway use this package; we need to go through the due diligence of getting this package vetted, licensed, and properly upstreamed into a separate repo. Is it possible we can just take the code from this tool and use it directly in our repo rather than referencing the tool as a dependency right now?